README.md

```diff
@@ -0,0 +1,399 @@
+# How to use Azure functions with MongoDB Atlas in Java
+
+> In this article we will learn how to use [MongoDB atlas](https://www.mongodb.com/atlas/database), a cloud database,
+> when you are getting started with [Azure functions](https://learn.microsoft.com/en-us/azure/azure-functions/functions-overview) 
+> in Java.
+
+## Introduction
+
+Cloud computing is one of the most discussed topics in the tech industry. Its ability to scale up and down infrastructure instantly, serverless
+apps are just a few benefits to start with. In this article, we are going write the function as a service(FaaS)e i.e. serverless function that 
+would interact with data via a database to produce meaningful results.    
```

Comment:
Let's make this paragraph the first paragraph. Also, consider adding a short example scenario in the introduction. The introduction is your hook and it will often determine who continues reading.